src/elm/Transaction/Summary.elm

```diff
@@ -90,12 +92,16 @@ type UpdateType
       -- ^Update the distribution of transaction fees
     | UpdateGASRewards
       -- ^Update the GAS rewards
-    | UpdateBakerStakeThreshold
-      -- ^Update the minimum stake that a baker needs to have to be able to bake
     | UpdateAddAnonymityRevoker
       -- ^Add a new anonymity revoker
-    | UpdateAddIdentityProvider -- ^Add a new identity provider
-
+    | UpdateAddIdentityProvider
+      -- ^Add a new identity provider
+    | UpdatePoolParameters
+      -- ^Update for pool parameters (previously baker stake threshold)
```

Comment:
I am not sure the comments about the "baker stake threshold" will make much sense beyond this PR. Consider removing them. (There are also a few further up).

Reply:
I'm hesitating to remove anything related to the protocol versions below 4 because the dashboard needs to support all protocol versions at the same time, even the comments. Additionally, these comments are copied from `Concordium.Types.Updates` without change, just like the type constructors themselves. This seems to be the common practice for all types mirroring the types in the consensus layer.